src/components/ReportWelcomeText.js

```diff
@@ -117,11 +126,17 @@ const ReportWelcomeText = (props) => {
                     <Text style={styles.textAlignCenter}>
                         {props.translate('reportActionsView.beginningOfChatHistory')}
                     </Text>
-                    {_.map(displayNamesWithTooltips, ({displayName, pronouns}, index) => (
+                    {_.map(displayNamesWithTooltips, ({
+                        displayName, pronouns, login, tooltip,
+                    }, index) => (
                         <Text key={displayName}>
-                            <Text style={[styles.textStrong]}>
-                                {displayName}
-                            </Text>
+                            <Tooltip text={tooltip}>
+                                <Pressable onPress={() => Navigation.navigate(ROUTES.getDetailsRoute(login))}>
+                                    <Text style={[styles.textStrong]}>
+                                        {displayName}
+                                    </Text>
+                                </Pressable>
+                            </Tooltip>
```

Comment:
```suggestion
                            <Tooltip text={tooltip} absolute>
                                    <Text style={[styles.textStrong]} onPress={() => Navigation.navigate(ROUTES.getDetailsRoute(login))}>
                                        {displayName}
                                    </Text>
                            </Tooltip>
```

Reply:
@parasharrajat I tried this but it doesn't work.

Reply:
Oh, I see. This is android. I was thinking web. 

So on Native, there is an extra wrapper on the tooltip, which seems completely unnecessary to me. Also, absolute implementation is not done on native (bcz it was not needed). 

Removing that wrapper `View` should fix this issue.

Reply:
@parasharrajat  yeah it works. @chirag should I go forward with it?

Reply:
Do test this carefully. Tooltip is a widely used component.

Reply:
Sure yes go forward with the change. As Rajat suggested do test this carefully. If its not working consistently everywhere then let's remove this change and keep it a the separate issue. 
If you can resolve it here I'll request for a price increase in the issue to make sure you are fairly compensated since this change is technically out of scope of your original issue.

Reply:
Made the changes and tested almost every place where we have tooltip, it didn't break anything.